src/stories/Button.stories.mdx

```diff
@@ -82,6 +82,78 @@ export const Template = args => <Button {...args} />;
 
 <br />
 
+### Rounded
+
+<Story
+  name="Rounded"
+  args={{
+    mode: 'rounded',
+    children: 'Read more'
+  }}
+>
+  {Template.bind({})}
+</Story>
+
+### Ghost
+
+<Story
+  name="Ghost"
+  args={{
+    mode: 'ghost',
+    children: 'Read more'
+  }}
+>
+  {Template.bind({})}
+</Story>
+
+### Link
+
+<Story
+  name="Link"
+  args={{
+    mode: 'link',
+    children: 'Submit'
+  }}
+>
+  {Template.bind({})}
+</Story>
+
+### Small
```

Comment:
All of these pills. I prefer the client defines themselves. They're too specific.